velox/expression/CastExpr.h

```diff
@@ -40,29 +40,25 @@ class CastOperator {
   /// @param input The flat or constant input vector
   /// @param context The context
   /// @param rows Non-null rows of input
-  /// @param nullOnFailure Whether this is a cast or try_cast operation
```

Comment:
nit: perhaps, clarify that per-row errors should be reported via context.setError, not thrown.